addons/web/static/src/scss/graph_view.scss

```diff
@@ -1,5 +1,6 @@
 .o_graph {
     .o_graph_svg_container {
+        direction: ltr#{"/*rtl:ignore*/"};
```

Comment:
@qsm-odoo We want to keep ltr direction forcefully on ```o_graph_svg_container``` element(as parent will have direction rtl in arabic or other rtl direction language).
About directive we used here: well ```/*rtl:ignore*/``` is enough to tell rtlcss to ignore it but in case of scss we need to pass it like ```#{/*rtl:ignore*/}```
https://github.com/MohammadYounes/rtlcss/issues/69
https://rtlcss.com/learn/usage-guide/value-directives/#tip